roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -30,6 +30,12 @@
       virsh -c qemu:///system -q
       domifaddr --source arp
       cifmw-{{ nic.host }} | grep "{{ nic.mac }}"
+      {% if cifmw_openshift_api_ip_address is defined %}
```

Comment:
aren't both set, meaning you'll want to chain the greps, with probably different address, and eventually fail with no address?